gengokit/httptransport/templates_test.go

```diff
@@ -201,7 +197,7 @@ func DecodeHTTPSumZeroRequest(_ context.Context, r *http.Request) (interface{},
 `
 	if got, want := str, desired; got != want {
```

Comment:
I'll probably keep objecting to this blind assignment of `got, want` in tests, *especially* in the case where the values under test are already contained in variables. I'm not going to choose it as my hill to make a final stand on, but it mostly seems like extra verbosity to me.